### DISCUSSION TITLE
Welcome to genshin-impact-script Discussions!

### DISCUSSION THREAD

**Discussion by phonowell** [phonowell/genshin-impact-script#79] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NuclearAck** [comment]
啊 怎么031 不支持CN了

**Reply by phonowell** [reply]
仍然支持

**Reply by NuclearAck** [reply]
启动后 游戏按键没反应... 是需要改哪里吗

**Reply by NuclearAck** [reply]
请问 GIS 和 GISS  有什么区别呢... 试了都不行

**Reply by B-Endorphin** [reply]
我也是没有反应

**Reply by B-Endorphin** [reply]
> 启动后 游戏按键没反应... 是需要改哪里吗

选择GIS的就行了

**Comment by Gptaqbc** [comment]
Is it just me or I can't find the Start.exe?

<<Go to the unpacked folder and double-click `start.exe` in it (you will be prompted if you want to apply administrator privileges, select **Apply**)>>

**Reply by phonowell** [reply]
The documentation is outdated and will be updated later
Your target is the only `.exe` file in the folder

**Reply by Amagashita** [reply]
Im not seeing any .exe file in the folder.  I tried to run the slim.ahk file though and idk if I'm messing up, but even when I setup the config file to link to where I have genshin, none of the script commands work.  Am I messing something up?

**Reply by Gursharn007** [reply]
I don't see an .exe either. How do I compile this?

**Reply by EchoesFromBeyond** [reply]
Just download the release on the right from the "Releases" section.

**Reply by EchoesFromBeyond** [reply]
Also make sure you get GIS and not GISS, not sure what the GISS is but it didn't work for me at all.

**Reply by EchoesFromBeyond** [reply]
@Gursharn007 / @Amagashita pinging so you get a notification about my comments. I will try to help you with it.

**Reply by Amagashita** [reply]
@EchoesFromBeyond wow thanks this worked.  I was downloading from source so that was my problem

**Reply by EchoesFromBeyond** [reply]
@Amagashita no worries, have fun, i love this thing :)

**Comment by xidecca** [comment]
Hello, I'm really new to this script stuff...can someone explain to me how to use it? Do I just run the .exe and then I can start using all the hotkeys?

**Reply by HotdogMassacre** [reply]
Yes.

**Reply by AC-V255** [reply]
我也是这个脚本的新手我下载了压缩包厚然后用管理员身份运行了gis.exe后该怎么做，要进游戏修改什么东西吗热键怎么使用

**Comment by Tianseen** [comment]


**Reply by phonowell** [reply]
what is the version of your `.exe`?
and post the content of the `config.ini`
